workbench-core/environments/src/interfaces/environmentTypeConfig.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import { z } from 'zod';
+
+// eslint-disable-next-line @rushstack/typedef-var
+export const EnvironmentTypeConfigParser = z.object({
+  id: z.string(),
+  type: z.string(),
+  description: z.string(),
+  name: z.string(),
+  createdAt: z.string(),
+  updatedAt: z.string(),
+  estimatedCost: z.string().optional(),
+  params: z.array(
+    z.object({
+      key: z.string(),
+      value: z.string()
+    })
+  )
+});
```

Comment:
Why not strict here?

Reply:
this is used to parse a ddbitem with more props than the ones we need to display, this is not used to parse requests